cajita/unit_test/CMakeLists.txt

```diff
@@ -81,6 +81,10 @@ set(SERIAL_TESTS
   Splines
   )
 
+if(NOT Kokkos_ENABLE_OPENMPTARGET) #FIXME_OPENMPTARGET
```

Comment:
Logic here is also different than the original cmake file. Is this on or off when OpenMP Target is on?

Reply:
Fixed